pkg/openwsn/include/board_info.h

```diff
@@ -31,6 +31,12 @@
 #include "periph_conf.h"
 #include "timex.h"
 
+#if defined __has_include
+#  if __has_include ("openwsn_defs.h")
+#    include "openwsn_defs.h"
+#  endif
```

Comment:
Isn't this include always needed or is this completely optional in OpenWSN?

Reply:
Optional, I just added it to override the definitions in `board_info.h` (a mandatory header in OpenWSN holding some board specific definitions or configurations).